### PR TITLE
Bug Fix: Rocky Linux 8.8-8.10 requires Python 3.11

### PR DESCRIPTION
**Pull request by BenjamenMeyer**
Thank you for submitting a pull request and becoming a contributor to Vega Strike's Build System Docker Images.

Please answer the following:

Code Changes:
- [ ] This is a documentation change only
- [X] CI Change

Purpose:
When updating the images the Python 8.8-8.10 images were changed from Python 3.11 Devel to just Python3 Devel which meant depending on Python 3.6.

Interestingly if Python 3.6 and 3.11 are installed then CMake will lock only Python 3.6. So we have to cleanup the image so it will build correctly too.

### CODE REVIEW

File: script/bootstrap [hunk 1]
```diff
@@ -1734,7 +1734,7 @@ function bootstrapOnRockyLinux ()
                           libpng-devel \
                           expat-devel \
                           gtk3-devel \
-                          python3-devel \
+                          python3.11-devel \
```
Comment (royfalk) [inline script/bootstrap:1737]: I was unable to find out if this is the right version. Rocky doesn't have a package management site as it is based on RHEL. RHEL has one but you apparently need a login.

**Review by royfalk — APPROVED** [review]
I'm fine trimming the versions to fewer. However, I don't know how to review the bootstrap code without going down a rabbit hole.

File: script/bootstrap (hunk 1 above)
Comment (BenjamenMeyer) [reply script/bootstrap:1737]: It's what we had for 8.8 originally and I mistakenly took it out. I did get it to compile once I added this back in and removed the other one as it seems cmake will find one and stop there.

Iirc there's a 3.12 too.

**Comment by BenjamenMeyer** [comment]
> I'm fine trimming the versions to fewer. However, I don't know how to review the bootstrap code without going down a rabbit hole. 

That's why I added the vs-image.sh tooling. So it was easy to get into the images. I've got another little rev to it that will make the base image and report local images too. So everything can be easy setup and done with little effort. Might need to document it better though.

**Comment by BenjamenMeyer** [comment]
Oh...and after merging we do a release to make the images for the CI system.

**Pull request merged**